openupgrade_scripts/scripts/base/14.0.1.3/noupdate_changes.xml

```diff
@@ -22,9 +22,6 @@
   <record id="ca" model="res.country">
     <field name="state_required">1</field>
   </record>
-  <record id="default_user" model="res.users">
```

Comment:
Thanks for the update! Please don't remove this entry but `<!--` uncomment it instead. That way it will be respected when the analysis is updated in the future.